strategies/mas/experiment.py

```diff
@@ -0,0 +1,97 @@
+import unittest
+
+import torch
+from torch.nn import CrossEntropyLoss
+from torch.optim import SGD
+
+from avalanche.evaluation.metrics import (
+    accuracy_metrics,
+    forgetting_metrics,
+    loss_metrics
+)
+from avalanche.training.plugins import EvaluationPlugin
+from models import MultiHeadVGGSmall
+from strategies.utils import create_default_args, get_average_metric
+from strategies.utils import get_target_result
+import avalanche as avl
+
+
+class MAS(unittest.TestCase):
+    """
+    Reproducing Memory Aware Synapses experiments from paper
+    "A continual learning survey: Defying forgetting in classification tasks"
+    by De Lange et al.
+    https://doi.org/10.1109/TPAMI.2021.3057446
+    """
+
+    def test_stinyimagenet(self, override_args=None, dataset_root=None):
```

Comment:
same here for `dataset_root` argument. This is just to enforce the same interface for all tests